src/main/java/eevee/Eevee.java

```diff
@@ -0,0 +1,161 @@
+package eevee;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Represents a task management program.
+ */
+public class Eevee {
+    private Ui ui;
+    private Storage storage;
+    private TaskList tasks;
+    private Parser parser;
+
+    /**
+     * Constructs an instance of Eevee with a specified storage file path.
+     *
+     * @param filePath The storage file path.
+     */
+    public Eevee(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+        this.tasks = new TaskList();
+        this.parser = new Parser();
+
+        try {
+            storage.loadTasks(tasks);
+        } catch (FileNotFoundException e) {
+            ui.printMessage("File not found!");
+        }
+    }
+
+    /**
+     * Starts the Eevee program, takes in user input and executes commands.
+     */
+    public void run() {
+        ui.printGreeting();
+
+        while (true) {
+            String input = ui.getInput();
+
+            try {
+                Parser.Command command = parser.parseCommand(input);
+
+                switch (command) {
```

Comment:
Using t, s, d, e as names in the entire switch statement can be a little troublesome to remember since there is an increasing number of single lettered variables. Would writing it in full be better?